js/manager.js

```diff
@@ -276,6 +286,14 @@ const DFPManager = Object.assign(new EventEmitter().setMaxListeners(0), {
     this.removeListener('slotRenderEnded', cb);
   },
 
+  attachSlotIsViewable(cb) {
+    this.on('impressionViewable', cb);
+  },
+
+  detachSlotIsViewable(cb) {
+    this.removeListener('impressionViewable', cb);
+  },
+
```

Comment:
this looks good!, just two small things:
1) can you please update the docs so to make mention of this new event and propType?
2) please revert changes made to packge.json.

thanks

Reply:
I just made the changes you requested. Thank you.